dumper/app/src/main/resources/hadoop-scripts/hive-version.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if command -v hive > /dev/null 2>&1 ; then
+  hive --version
+fi
```

Comment:
Can you please add missing new line at the end of this shell script and others?

Reply:
Done.